SecurityService.OpenIdConnect.IntegrationTests/Common/DockerHelper.cs

```diff
@@ -121,9 +247,20 @@ public Hooks(IObjectContainer objectContainer)
         public async Task BeforeScenario()
         {
             ChromeOptions options = new ChromeOptions();
-            options.AddArguments("--window-size=1920,1080");
-            options.AddArguments("--start-maximized");
-            options.AddArguments("--headless");
+            //options.AddArguments("--window-size=1920,1080");
+            //options.AddArguments("--start-maximized");
+            options.AddArguments("--disable-gpu");
+            options.AddArguments("--no-sandbox");
+            options.AddArguments("--disable-dev-shm-usage");
+            //options.AddArguments("--browser.enabled_labs_experiments=same-site-by-default-cookies=disabled");
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/stuart_ferguson1/SecurityService/pullRequest?prid=6144581)